src/main/java/ca/on/oicr/gsi/dimsum/service/CaseService.java

```diff
@@ -325,73 +324,73 @@ public TableData<ProjectSummaryRow> getProjectSummaryRows(String name) {
     ProjectSummaryRow pendingWork = new ProjectSummaryRow.Builder()
         .title("Pending Work")
         .extraction(
-            new ProjectSummaryField.Builder().count(projectSummary.getExtractionPendingCount())
-                .filterKey("PENDING").filterValue("Extraction").build())
+            new ProjectSummaryField(projectSummary.getExtractionPendingCount(),
+                "PENDING", "Extraction"))
```

Comment:
Use the enums instead of duplicating the Strings. This way, it's easy to find usage, rename things in the future, etc.

"PENDING" -> `CaseFilterKey.PENDING.name()`
"Extraction" -> `PendingState.EXTRACTION.getLabel()`